blog/introduction_to_data_models_in_cloud_datastore/blog.py

```diff
@@ -43,7 +43,7 @@ def create_user(ds, username, profile):
 
```

Comment:
nit: Need header adjustment on this file:
`# Copyright 2015, Google, Inc.`
to
`# Copyright 2015 Google, LLC.`